shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -921,6 +927,42 @@ - (void)waitForFirstFrame:(NSTimeInterval)timeout
   });
 }
 
+- (FlutterEngine*)spawnWithEntrypoint:(NSString*)entrypoint {
```

Comment:
add a comment? if you add new features to createShell, please make sure it's mirrored here, and vice versa. 

Could add some contextual reasoning too. We could merge it into the same API except the Dart APIs only support creating Isolates from other Isolates. Here's a dart SDK github issue.

Reply:
A change in called functions will often require a change in caller functions.  I'm not sure it's worth point that out.  I don't know code comments is the right place for design decisions.  If you want to suggest something specific we can put it in there.